apps/web-client/src/lib/stores/inventory/__tests__/table_content.test.ts

```diff
@@ -24,4 +25,41 @@ describe('tableContentStore', () => {
 			totalItems: 0
 		} as PaginationData);
 	});
+	test('should create display store from db and entries streams', async () => {
+		const db = await newTestDB();
+		await db.upsertBook({
+			isbn: '0007149522',
+			title: 'The Age of Wonder',
+			authors: ['Richard Holmes'],
+			publisher: 'HarperCollins UK',
+			year: '2008',
+			price: 69.99
+		});
+		const note = await db.warehouse().note('note-1').create();
+		const books = await db.getBooks(['0007149522']);
+		await note.update({
```

Comment:
`note.update` doesn't exist (I obviously have to fix type system...). What you want to do is do `note.addVolumes` (and pass an array of entries)

Reply:
On a semi-related note: you can always check that the note was updated properly by doing `console.log(note)` (thanks to our carefully created class, only the note fields will get logged to the console, without the methods and private values...should make for easier debugging)